### PR TITLE
Parse JavaScript coverage

### LINKED ISSUE #54: Parse JavaScript coverage too
Right now, we are only parsing C++ coverage data. We need to parse JavaScript coverage too.

Here's how we are doing it in mozilla-central: https://dxr.mozilla.org/mozilla-central/rev/763f30c3421233a45ef9e67a695c5c241a2c8a3a/testing/mozharness/mozharness/mozilla/testing/codecoverage.py#243

In particular the relevant parts are:
https://dxr.mozilla.org/mozilla-central/rev/763f30c3421233a45ef9e67a695c5c241a2c8a3a/testing/mozharness/mozharness/mozilla/testing/codecoverage.py#252-255
and:
https://dxr.mozilla.org/mozilla-central/rev/763f30c3421233a45ef9e67a695c5c241a2c8a3a/testing/mozharness/mozharness/mozilla/testing/codecoverage.py#280

`lcov_rewriter` is a Python module that is in mozilla-central source code, but we can download it from an artifact (like we do for the coverage build itself).

### PR DESCRIPTION
**Pull request by rhcu**
Fixes #54

### CODE REVIEW

File: crawler.py [hunk 1]
```diff
@@ -229,13 +229,23 @@ def run_all(driver, data_folder):
...
     run_all(driver, data_folder)
 
+    sys.path.insert(0, 'tools/mozbuild/codecoverage')
+
+    from lcov_rewriter import LcovFileRewriter
+
+    jsvm_output_file = 'jsvm_lcov_output.info'
+    jsvm_files = [os.path.join(jsvm_dir, e) for e in os.listdir(jsvm_dir)]
+    rewriter = LcovFileRewriter(os.path.join('tools', 'chrome-map.json'))
+    rewriter.rewrite_files(jsvm_files, jsvm_output_file, '')
```
Comment (marco-c) [inline crawler.py]: We should remove the jsvm_files after rewriting them into jsvm_output_file, and remove jsvm_output_file at the end (when we also remove code-coverage-gcda.zip.

Comment (rhcu) [reply crawler.py]: `jsvm_files` are in the temporary directory created with context manager - aren't they deleted automatically by OS?

Comment (marco-c) [reply crawler.py]: Yes, I forgot `jsvm_dir` was a temporary directory.

File: crawler.py [hunk 2]
```diff
@@ -229,13 +229,23 @@ def run_all(driver, data_folder):
     data_folder = str(uuid.uuid4())
     run_all(driver, data_folder)
 
+    sys.path.insert(0, 'tools/mozbuild/codecoverage')
+
+    from lcov_rewriter import LcovFileRewriter
```
Comment (marco-c) [inline crawler.py]: We can actually import this at the beginning of the file

**Pull request merged**